puppet/manifests/default.pp

```diff
@@ -17,6 +52,34 @@
   }
 
   include docker_registry
+
+  include glusterfs_common
+
+  gluster::peer { 'standalone.seed-stack.local':
+    require => Package['glusterfs-server'],
+  }
+
+  # We need at least two replicas, so they both have to live on the same node
+  # in the single-machine setup.
+  file { '/data/brick2':
+    ensure  => 'directory',
+    require => File['/data'],
+  }
+
+  gluster::volume { 'data1':
+    replica => 2,
+    force   => true,
```

Comment:
Without that gluster won't let us use the root filesystem for brick storage. I'll add a comment.